seqr/management/commands/lift_project_to_hg38.py

```diff
@@ -11,7 +11,7 @@
 from seqr.views.utils.json_to_orm_utils import update_model_from_json
 from seqr.views.utils.orm_to_json_utils import get_json_for_saved_variants
 from seqr.views.utils.variant_utils import reset_cached_search_results
-from seqr.utils.elasticsearch.utils import get_es_variants_for_variant_tuples, get_single_es_variant
+from seqr.utils.search.utils import get_es_variants_for_variant_tuples, get_single_es_variant
```

Comment:
I think all the `es` related utils should be under the `elasticsearch` sub-folder.

Reply:
thats the plan, so  we would refactor `get_es_variants_for_variant_tuples` to `get_variants_for_variant_tuples` and then move the es-specific parts of the function into the nested `elasticsearch` folder. But to keep the diff on this sane, I started by just moving the files with no changes, and will have a leter follow up PR that splits up the logic